test/TestHelper.cpp

```diff
@@ -88,6 +87,35 @@ ImportTest::ImportTest(json_spirit::mObject& _o, bool isFiller):
 	}
 }
 
+json_spirit::mObject& ImportTest::makeAllFieldsHex(json_spirit::mObject& _o)
+{
+	for (json_spirit::mObject::iterator it = _o.begin(); it != _o.end(); it++)
+	{
+		string key = (*it).first;
+		if (key == "to")
+			continue;
+
+		string str;
+		json_spirit::mValue value = (*it).second;
+
+		if (value.type() == json_spirit::int_type)
+			str = toString(value.get_int());
+		else if (value.type() == json_spirit::str_type)
+			str = value.get_str();
+		else continue;
+
+		_o[key] = (str.substr(0, 2) == "0x" ||
+					str.find("a") != string::npos ||
```

Comment:
if a value has only numbers in it, it could still be a hex. Therefore just check for "0x" as you do, and the rest are exceptions such as "to", see comment above.